launcher/container_runner.go

```diff
@@ -150,6 +155,13 @@ func NewRunner(ctx context.Context, cdClient *containerd.Client, token oauth2.To
 		return nil, &RetryableError{fmt.Errorf("cannot get hostname: [%w]", err)}
 	}
 
+	var rlimits []specs.POSIXRlimit
```

Comment:
nit: can you do `rlimits := []specs.POSIXRlimit{specs.POSIXRlimit{...}}`

Reply:
Done